web-frontend/src/pages/Assets/AssetsImportModal.tsx

```diff
@@ -23,16 +63,22 @@ export function AssetsImportModal({ ...props }) {
       <PopupModalContent>
         <div className="mas-body2 pb-10">
           <p className="mb-6">{Intl.t('assets.import-subtitle')}</p>
-          <Input customClass="mb-2" />
-          <Button
-            type="submit"
-            onClick={() => {
-              console.log('submit');
-            }}
-            preIcon={<FiPlus size={24} />}
-          >
-            {Intl.t('assets.add')}
-          </Button>
+          <form onSubmit={handleSubmit}>
+            <Input
+              placeholder={'Token Address'}
+              value={tokenAddress}
+              name="tokenAddress"
+              onChange={(e) => setTokenAddress(e.target.value)}
+              error={inputError?.address}
+            />
```

Comment:
```suggestion
            <Input
              placeholder={'Token Address'}
              default-value=""
              name="tokenAddress"
              error={inputError?.address}
            />
```

You use an uncontrolled form here. So you can remove `value` & `onChange` variables. 😉